x-pack/legacy/plugins/remote_clusters/index.ts

```diff
@@ -7,8 +7,6 @@
 import { Legacy } from 'kibana';
```

Comment:
I think if the goal of this contribution is to complete the NP migration we should strip this file down to the bare minimum required to keep functional parity with legacy. For instance:

- Migrating `config` and `injectDefaultVars` see https://github.com/elastic/kibana/blob/master/src/core/MIGRATION.md#configure-plugin
- Similarly `configPrefix` can be removed if we define it in `kibana.json` I believe
- If we use the new platform management service for registering the plugin can we remove `managementSections`?
- Is `isEnabled` being used somewhere else?

Reply:
👍 I was planning on addressing these items in a separate PR as part of the `public` migration. 

Also, `isEnabled` is being used by CCR so I think this needs to stay as is for now.